packages/flutter_tools/templates/module/ios/library/Flutter.tmpl/podhelper.rb.tmpl

```diff
@@ -112,6 +110,8 @@ def flutter_root
   raise "FLUTTER_ROOT not found in #{generated_xcode_build_settings_path}. Try deleting Generated.xcconfig, then run flutter pub get"
 end
 
+require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)
```

Comment:
any reason why this shouldn't be at the top of the file?

Reply:
This line of code must be placed after the `flutter_root` method, otherwise the following error will be reported.

```shell
❯ pod install

[!] Invalid `Podfile` file: undefined local variable or method `flutter_root' for main:Object

require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)
                                                                                     ^^^^^^^^^^^^.
```

It is also used like this in `Podfile-ios-objc`.

https://github.com/flutter/flutter/blob/67457e669f79e9f8d13d7a68fe09775fefbb79f4/packages/flutter_tools/templates/cocoapods/Podfile-ios-objc#L13-L26

Reply:
Ahh I see, makes sense